Cannot restart a previous session

When trying to restart a previous ruruku session (e.g. `ruruku start --session-name foobar` with a `foobar.yaml` present) the server won't start. There seems to be an issue in the YAML unmarshall process:
```
$ ruruku start --session-name ruruku-20181128-083215 testsuite.yaml
panic: reflect: reflect.Value.Set using unaddressable value [recovered]
        panic: reflect: reflect.Value.Set using unaddressable value

goroutine 1 [running]:
gopkg.in/yaml%2ev2.handleErr(0xc00014f938)
        /home/travis/gopath/src/gopkg.in/yaml.v2/yaml.go:249 +0x9a
panic(0x8db9c0, 0xc000221a40)
        /home/travis/.gimme/versions/go1.11.linux.amd64/src/runtime/panic.go:513 +0x1b9
reflect.flag.mustBeAssignable(0x16)
        /home/travis/.gimme/versions/go1.11.linux.amd64/src/reflect/value.go:234 +0x157
reflect.Value.Set(0x900740, 0x0, 0x16, 0x900740, 0xc0000b3f20, 0x16)
        /home/travis/.gimme/versions/go1.11.linux.amd64/src/reflect/value.go:1367 +0x2f
gopkg.in/yaml%2ev2.(*decoder).prepare(0xc0000b5f80, 0xc00021ed90, 0x900740, 0x0, 0x16, 0x0, 0x40e71d, 0xc0000b0170, 0xc000172d80)
        /home/travis/gopath/src/gopkg.in/yaml.v2/decode.go:302 +0x2bf
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0000b5f80, 0xc00021ed90, 0x900740, 0x0, 0x16, 0x1)
        /home/travis/gopath/src/gopkg.in/yaml.v2/decode.go:324 +0x74
gopkg.in/yaml%2ev2.(*decoder).document(0xc0000b5f80, 0xc00021ed20, 0x900740, 0x0, 0x16, 0xc00014f668)
        /home/travis/gopath/src/gopkg.in/yaml.v2/decode.go:344 +0x7a
gopkg.in/yaml%2ev2.(*decoder).unmarshal(0xc0000b5f80, 0xc00021ed20, 0x900740, 0x0, 0x16, 0x600)
        /home/travis/gopath/src/gopkg.in/yaml.v2/decode.go:320 +0x1d3
gopkg.in/yaml%2ev2.unmarshal(0xc000222000, 0x36c, 0x56c, 0x900740, 0x0, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/gopkg.in/yaml.v2/yaml.go:148 +0x3a1
gopkg.in/yaml%2ev2.Unmarshal(0xc000222000, 0x36c, 0x56c, 0x900740, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/gopkg.in/yaml.v2/yaml.go:81 +0x58
github.com/32leaves/ruruku/pkg/rurukulib/server.LoadFileStorage(0xc000217660, 0x1b, 0xc000172a80, 0xc0001f8ea0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/32leaves/ruruku/pkg/rurukulib/server/storage.go:57 +0x145
github.com/32leaves/ruruku/pkg/rurukulib/server.LoadSessionOrNew(0x7fffe2905f93, 0x16, 0xc000172a80, 0x0, 0x0, 0x0)
        /home/travis/gopath/src/github.com/32leaves/ruruku/pkg/rurukulib/server/session.go:23 +0x240
github.com/32leaves/ruruku/pkg/rurukulib/server.Start(0xc00014fcb0, 0xc000172a80, 0x7fffe2905f93, 0x16, 0x0, 0xc00014fcc8)
        /home/travis/gopath/src/github.com/32leaves/ruruku/pkg/rurukulib/server/server.go:63 +0x79
github.com/32leaves/ruruku/cmd.glob..func3(0x1391480, 0xc0000e89b0, 0x1, 0x5)
        /home/travis/gopath/src/github.com/32leaves/ruruku/cmd/start.go:34 +0x17d
github.com/spf13/cobra.(*Command).execute(0x1391480, 0xc0000e8960, 0x5, 0x5, 0x1391480, 0xc0000e8960)
        /home/travis/gopath/src/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0x1391220, 0xc0000a3660, 0xf, 0xc0000af7b0)
        /home/travis/gopath/src/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0x1391220, 0x407250, 0xc000098058)
        /home/travis/gopath/src/github.com/spf13/cobra/command.go:800 +0x2b
github.com/32leaves/ruruku/cmd.Execute()
        /home/travis/gopath/src/github.com/32leaves/ruruku/cmd/root.go:29 +0x2d
main.main()
        /home/travis/gopath/src/github.com/32leaves/ruruku/main.go:20 +0x20
```

tested in v0.1.3

--- a/pkg/rurukulib/server/storage.go
+++ b/pkg/rurukulib/server/storage.go
@@ -54,7 +54,7 @@ func LoadFileStorage(fn string, suite *protocol.TestSuite) (*FileStorage, error)
         return nil, err
     }
 
-    if err := yaml.Unmarshal(fc, r.run); err != nil {
+    if err := yaml.Unmarshal(fc, &r.run); err != nil {
         log.WithField("filename", fn).WithError(err).Error("Error while restoring storage")
         return nil, err
     }

